lightning/src/ln/channel.rs

```diff
@@ -4771,6 +4760,33 @@ pub(super) struct DualFundingChannelContext {
 	pub our_funding_inputs: Vec<(TxIn, TransactionU16LenLimited)>,
 }
 
+impl DualFundingChannelContext {
+	/// Obtain prev outputs for each supplied input and matching transaction.
+	/// Can error when there a prev tx does not have an output for the specified vout number.
```

Comment:
Should change to 

`/// Will error when a prev tx does not have an output for the specified vout.`

Reply:
Changed